imports/plugins/core/checkout/client/components/cancelOrderButton.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Components } from "@reactioncommerce/reaction-components";
+
+const CancelOrderButton = (props) => {
+  const { order, orderStatus, onCancelOrderClick } = props;
+  return (
+    <div>
+      {
+        (orderStatus === "new" || orderStatus === "coreOrderWorkflow/canceled") &&
+      <Components.Button
+        status="primary"
+        disabled={orderStatus === "coreOrderWorkflow/canceled"}
+        buttonType="submit"
+        bezelStyle="solid"
+        onClick={() => { onCancelOrderClick(order); }}
+        label={orderStatus === "coreOrderWorkflow/canceled" ? "Cancelled" : "Cancel Order"}
```

Comment:
You can assign `orderStatus === "coreOrderWorkflow/canceled"` to a variable at the top since it is used 3 times and means the same thing